packages/theme-predictive-search-component/README.md

```diff
@@ -0,0 +1,57 @@
+## Description
+
+A component that controls the state of a predictive search UI.
+
+## Example
+
+```js
+const predictivesearch = new PredictiveSearch({
+  selectors: {
+    input: '[data-predictive-search-input="header"]',
+    reset: '[data-predictive-search-reset="header"]',
+    result: '[data-predictive-search-result="header"]'
+  },
+  resultTemplateFct: data => {
+    return `
+      <div class="predictive-search">
+        <ul class="predictive-search__list">
+          ${products.map(
+            product => `
+            <li>
+              <img src="${product.image}" alt="${product.title}" />
+              <span>${produtc.title}</span>
+            </li>
+          `
+          )}
+        </ul>
+      </div>
+    `;
+  },
+  onInputFocus: nodes => {
+    // You can get a reference to active target
+    console.log(nodes.input.id); //-> input
+    console.log(nodes.submit.id); //-> submit
+    console.log(nodes.result.id); //-> result
+  },
```

Comment:
I find this confusing: Does this callback only fires when the input is focused? or does it callback when either input, submit, or result focuses?

What is the purpose to have this event proxy if it's only giving a callback on input focus? Could these be solved by simply having a bindEvent callback so that theme developers can do whatever they need to do, be it focus, keyup, blur, touch, ... etc browser events?